website/content/partials/lease-count-quota-upgrade.mdx

```diff
@@ -0,0 +1,5 @@
+### Lease Count Quota Invalidations on DR Secondaries Fixed
+
+Lease count quota invalidation causes DR Secondaries to panic and experience
+a hard shutdown. This issue exists prior to Vault 1.6.6, and 1.7.4. It has been
```

Comment:
```suggestion
a hard shutdown. This issue exists prior to Vault 1.6.6, and 1.7.4. It is
```

Reply:
It won't quite be released when this PR merges, so we can leave the tense ambiguous this way.